web app/src/test/java/de/tinf15b4/kino/cucumber/SpringTestConfig.java

```diff
@@ -57,8 +58,11 @@ private static void startServer() throws Exception {
         pb.redirectInput(ProcessBuilder.Redirect.PIPE);
         pb.environment().put("SMARTCINEMA_DATA_API_LISTEN_ON", "" + port);
         pb.environment().put("SMARTCINEMA_DATA_API_KEEPALIVE_PIPE", "yeah");
+        pb.environment().put("spring.datasource.url", "jdbc:h2:file:~/smartCinemaDataBase/smartcinemaTest");
+        pb.environment().put("spring.jpa.hibernate.ddl-auto", "create-drop");
```

Comment:
Can't we use an in-memory database for the tests? Then we don't have to deal with the file